### PR TITLE
UX smooth scrolling. Filters form template.

### PR DESCRIPTION
**Pull request by junior1q94**

### CODE REVIEW

File: paradise_papers_search/static/css/search.css [hunk 1]
```diff
@@ -1,35 +1,28 @@
...
+#search-tabs {
+  margin-bottom: -50px;
+}
 
-/***************************************************************************
- * Results Search
- ***************************************************************************/
+#search-results {
+  max-width: 1280px;
+}
```
Comment (oscarpolanco) [inline paradise_papers_search/static/css/search.css]: Github will be happy with a extra line

File: paradise_papers_search/static/js/smoothscroll.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            // Prevent default anchor click behavior
+            event.preventDefault();
+            // Store hash
+            var hash = this.hash;
+            // Using jQuery's animate() method to add smooth page scroll
+            // The optional number (800) specifies the number of milliseconds it takes to scroll to the specified area
+            $('html, body').animate({
+                scrollTop: $(hash).offset().top
+            }, 800, function(){
+                console.log($(hash).offset().top);
```
Comment (oscarpolanco) [inline paradise_papers_search/static/js/smoothscroll.js]: Do we need this `console.log`? 😅 
Can we delete this?

File: paradise_papers_search/templates/index.html [hunk 3]
```diff
@@ -3,22 +3,34 @@
...
-              </select>
-              <select class="form-control col" id="inputGroupSelect01">
-                <option selected>All Countries</option>
-                <option value="1">One</option>
-                <option value="2">Two</option>
-                <option value="3">Three</option>
-              </select>
+              <div class="input-group mb-3 col-md-6">
+                <div class="input-group-prepend">
+                  <span class="input-group-text" id="">By jurisdiction: </span>
```
Comment (oscarpolanco) [inline paradise_papers_search/templates/index.html]: Extra space here

File: paradise_papers_search/templates/index.html [hunk 4]
```diff
@@ -3,22 +3,34 @@
...
+                </div>
+                <select class="form-control" id=""
+                  data-bind="options: _jurisdictionList,
+                             optionsCaption: 'All Jurisdictions...',
+                             value: _filters.jurisdiction">
+                </select>
+              </div>
+              <div class="input-group mb-3 col-md-6">
+                <div class="input-group-prepend">
+                  <span class="input-group-text" id="">By Country: </span>
```
Comment (oscarpolanco) [inline paradise_papers_search/templates/index.html]: Extra space here 😅

**Review by raulgomezc — APPROVED** [review]
LGTM

**Review by oscarpolanco — APPROVED** [review]
Minors comments
LGTM

**Review by mariamera — APPROVED** [review]

**Pull request merged**